src/models/network.js

```diff
@@ -189,7 +189,7 @@ Network.prototype.validate = function (client) {
 Network.prototype.createIrcFramework = function (client) {
 	this.irc = new IrcFramework.Client({
 		version: false, // We handle it ourselves
-		outgoing_addr: Helper.config.bind,
+		outgoing_addr: client.config.bind !== "undefined" ? client.config.bind : Helper.config.bind,
```

Comment:
Don't pass in undefined as string. Make it default to `null` (not string).

The prompt also doesn't really explain anything "Outgoing address?" is just ambiguous.